core/browser.vala

```diff
@@ -474,7 +474,9 @@ namespace Midori {
         }
 
         void about_activated () {
-            new About (this).show ();
+            var about = new About (this);
+            about.run ();
+            about.hide ();
```

Comment:
Please change the last call to `.close ()`, so we don't keep one or more hidden about dialogs in memory.

Reply:
As recommended by @kalikiana the last commit contains the fix which replace `.hide ()` with `.close ()`.